topiary-cli/src/main.rs

```diff
@@ -120,6 +107,11 @@ async fn run() -> CLIResult<()> {
                 results.remove(0)??
             } else if results
                 .iter()
+                .inspect(|r| match r {
+                    Err(e) => print_error(&e),
+                    Ok(Err(e)) if !e.benign() => print_error(&e),
```

Comment:
This is actually a bug because `.any` will be lazily evaluated thus only the first error will be printed, this is fixed in the check https://github.com/tweag/topiary/pull/1094 by using `.count()` instead of `.any()`:

https://github.com/mkatychev/topiary/blob/ad580dded8533f76aa73853e32e5c3bc9fc81533/topiary-cli/src/io.rs#L488-L497